src/main/java/com/softserveinc/Task03.java

```diff
@@ -1,11 +1,33 @@
 package com.softserveinc;
 
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class Task03 {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
-        // TODO: Write your code here
+        System.out.print("Write how many ml of water the coffee machine has:\n" + ">>> ");
+            int CupsFromWater = scanner.nextInt() / 200; //
```

Comment:
Local variables should have names in camelCase style. Instead CupsFromWater you should write cupsFromWater